library/lua/gui/widgets/text_area/text_area_content.lua

```diff
@@ -0,0 +1,677 @@
+local gui = require('gui')
+local common = require('gui.widgets.common')
+local Widget = require('gui.widgets.widget')
+local WrappedText = require('gui.widgets.text_area.wrapped_text')
+local HistoryStore = require('gui.widgets.text_area.history_store')
+
+local CLIPBOARD_MODE = {LOCAL = 1, LINE = 2}
+local HISTORY_ENTRY = HistoryStore.HISTORY_ENTRY
+
+TextAreaContent = defclass(TextAreaContent, Widget)
+
+TextAreaContent.ATTRS{
+    text = '',
+    text_pen = COLOR_LIGHTCYAN,
+    ignore_keys = {'STRING_A096'},
+    pen_selection = COLOR_CYAN,
+    on_text_change = DEFAULT_NIL,
+    on_cursor_change = DEFAULT_NIL,
+    enable_cursor_blink = true,
+    debug = false,
+    one_line_mode = false,
+    history_size = 10,
```

Comment:
the internet does not seem to have a good consensus of what a 'good' default history size is, but 10 seems low to me. How about 20 (or leave the attribute as `DEFAULT_NIL` and use the HistoryStore default of 25)?